solution/test/test-has-duplicates.js

```diff
@@ -0,0 +1,19 @@
+const hasDuplicates = require('../src/has-duplicates');
+const assert = require('assert');
+
+it("test when no duplicates", () => {
+    const numbers = [3, 2, 1, 4, 7, 6, 5];
+    assert(hasDuplicates(numbers) == false);
+});
+
+it("test when duplicates", () => {
```

Comment:
How will the following test case affect your solution?  ?
```
[1,1,4,4]
```

Reply:
Ah, my solution is naive, should have thought more thoroughly. Will fix.